cmd/yace/main.go

```diff
@@ -207,15 +215,15 @@ func (s *scraper) scrape(ctx context.Context, cache exporter.SessionCache) {
 	defer sem.Release(1)
 
 	newRegistry := prometheus.NewRegistry()
-	for _, counter := range []prometheus.Counter{exporter.CloudwatchAPICounter, exporter.CloudwatchAPIErrorCounter, exporter.CloudwatchGetMetricDataAPICounter, exporter.CloudwatchGetMetricStatisticsAPICounter, exporter.ResourceGroupTaggingAPICounter, exporter.AutoScalingAPICounter, exporter.ApiGatewayAPICounter, exporter.TargetGroupsAPICounter} {
+	for _, counter := range []prometheus.Counter{exporter.CloudwatchAPICounter, exporter.CloudwatchAPIErrorCounter, exporter.CloudwatchGetMetricDataAPICounter, exporter.CloudwatchGetMetricStatisticsAPICounter, exporter.ResourceGroupTaggingAPICounter, exporter.AutoScalingAPICounter, exporter.ApiGatewayAPICounter, exporter.TargetGroupsAPICounter, exporter.DmsAPICounter} {
```

Comment:
I _think_ we're missing `exporter.Ec2APICounter` here (should be 10 in total).

Reply:
Great catch! Perhaps I should do a follow up to export an array which contains all of them instead to help avoid this.